smart-contracts/test/Unlock/UnlockProxy.js

```diff
@@ -2,28 +2,28 @@ const zos = require('zos-lib')
 const Unlock = artifacts.require('Unlock')
 const UnlockTestV2 = artifacts.require('UnlockTestV2')
 const UnlockTestV3 = artifacts.require('UnlockTestV3')
-const AdminUpgradeabilityProxy = zos.Contracts.getFromNodeModules('zos-lib', 'AdminUpgradeabilityProxy')
+const TestHelper = require('zos').TestHelper
```

Comment:
Probably cleaner to have a top level
`const Zos = require('zos')`
and then
` const TestHelper = Zos.TestHelper`

Reply:
np